app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -13,6 +14,15 @@ const RATE_LIMIT_TYPES = {
   NON_RATE_LIMITED: 'non_rate_limited',
 };
 
+/**
+ * These types correspond to the UI Customization presented to the user. Right
+ * now there is only one UI customization option, but more could be added in
+ * the future.
+ */
+const UI_CUSTOMIZATION_TYPES = {
+  SIWE: 'sign_in_with_ethereum',
+};
```

Comment:
because we [plan to expand this option (internal)](https://consensys.slack.com/archives/G01FXS64DFA/p1659649733910519?thread_ts=1659542477.435949&cid=G01FXS64DFA), I recommend moving this `const` to `shared/constants/metametrics.js` since `trackEvent` may be called throughout the codebase.

another suggestion is we could start adding a const to keep track of custom keys. maybe something like:
```
const METAMETRIC_KEY = {
  UI_CUSTOMIZATIONS: `ui_customizations`,
};

const METAMETRIC_KEY_OPTIONS = {
  [METAMETRIC_KEY.UI_CUSTOMIZATIONS]: {
    SIWE: 'sign_in_with_ethereum',
  },
};
```

Reply:
Made this change! I would recommend reviewing the comment describing the option, I was unsure how to describe it. @digiwand